litellm/main.py

```diff
@@ -1420,6 +1422,49 @@ def completion(
                     original_response=response,
                 )
             response = response
+
+        elif custom_llm_provider == "notdiamond":
+            notdiamond_key = (
+                api_key
+                or litellm.notdiamond_key
+                or get_secret("NOTDIAMOND_API_KEY")
+                or litellm.api_key
+            )
+
+            api_base = (
+                api_base
+                or litellm.api_base
+                or get_secret("NOTDIAMOND_API_BASE")
+                # TODO: confirm?
+                or "https://not-diamond-server.onrender.com/v2/optimizer/router"
```

Comment:
Let's update this as previously discussed!

Reply:
Updated to `/modelSelect`